doc/releases/changelog-dev.md

```diff
@@ -4,12 +4,21 @@
 
 <h3>New features since last release</h3>
 
-* The `qml.fourier.advanced_spectrum` function extends the former `qml.fourier.spectrum` function
+* The `qml.fourier.qnode_spectrum` function extends the former `qml.fourier.spectrum` function
   and takes classical processing of QNode arguments into account.
   The frequencies are computed per (requested) QNode argument instead
   of per gate `id`. The gate `id`s are ignored.
   [(#1681)](https://github.com/PennyLaneAI/pennylane/pull/1681)
 
+  The updated version of `qml.fourier.spectrum`
+  behaves differently in three ways:
```

Comment:
This sentence doesn't make it explicit to me that:

- `qml.fourier.qnode_spectrum` is a new function
- Calling `qml.fourier.spectrum` is now an alias to this new function

Reply:
I added the first point but the second one is in the second entry of the changelog, no? Should I repeat it here in short?

Reply:
Okay, I removed the three-point comparison now and mention that a comparison to `circuit_spectrum` also can be found in the docstring. :)